lib/ui/painting/flutter_rtree_unittests.cc

```diff
@@ -0,0 +1,90 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/lib/ui/painting/flutter_rtree.h"
+#include "flutter/testing/testing.h"
+#include "flutter/testing/thread_test.h"
+
+#include "third_party/skia/include/core/SkCanvas.h"
+#include "third_party/skia/include/core/SkPictureRecorder.h"
+
+namespace flutter {
+namespace testing {
+
+using FlutterRTreeTest = ThreadTest;
```

Comment:
Not sure why this needs to be a thread test fixture. We aren't use threads here at all.